var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -435,12 +435,12 @@ def filter_compilers(self):
                 filter_file(env["CXX"], "CC", libtool_makefile, **kwargs)
 
         # nrnmech_makefile exists in both cmake and autotools builds
-        filter_file('CC = {}'.format(env["CC"]),
-                    'CC = {}'.format(cc_compiler),
+        filter_file('CC = %s' % env["CC"],
+                    'CC = %s' % cc_compiler,
                     nrnmech_makefile,
                     **kwargs)
-        filter_file('CXX = {}'.format(env["CXX"]),
-                    'CXX = {}'.format(cxx_compiler),
+        filter_file('CXX = %s' % env["CXX"],
+                    'CXX = %s' % cxx_compiler,
```

Comment:
Why????? `%` has to die!¡
```suggestion
                    'CXX = {0}'.format(cxx_compiler),
```

Reply:
I thought that there was an issue about that with travis.. I will fix it and change it back to `format`. Thanks 🙂

Reply:
Thank you! `{}` was introduced in 2.7, but `{0}` et al are ok with Python 2.6, which is what the CI should test…

Reply:
I see.. Thank you for the explanation! 🙂